polyfill/lib/ecmascript.mjs

```diff
@@ -600,7 +600,7 @@ export const ES = ObjectAssign({}, ES2019, {
     if (millis || parts.length) parts.unshift(`000${millis || 0}`.slice(-3));
     let secs = `00${seconds}`.slice(-2);
     let post = parts.length ? `.${parts.join('')}` : '';
-    return `${secs}${post}`;
+    return `:${secs}${post}`;
```

Comment:
I'm not quite sure about this one. On one hand, sure, it's common to literally every place this operation is called. On another, shouldn't the string interpolator in the calling function take care of the `:`?